framework/doc/content/automatic_differentiation/index.md

```diff
@@ -94,7 +94,7 @@ Mathematica; however, for functions of any
 complexity the resulting gradient expressions can take up to several pages and
 can be quite difficult to translate from notebook to
 code. An alternative to numerical and symbolic
-differentiation is [!ac](AD), which applies the chain rule to
+differentiation is automatic differentiation, [!ac](AD), which applies the chain rule to
```

Comment:
this will create a redundancy in the rendered html page

Reply:
Ah! So that's what [!ac] does! It fetches from `acronyms.yml`... But how does it decide when to expand?